### PR TITLE
Dump Offload:Reset Dump OffloadUri as part of clean up

### PR DESCRIPTION
**Pull request by raviteja-b**
Recently its decided that Dump manager gets know if offload is in
progress based on OffloadUri

If OffloadUri value is set, dump manager rejects other dump offload requests
This PR resets OffloadUri property to handle error scenarios.

### CODE REVIEW

**Review by sunharis — APPROVED** [review]

**Comment by raviteja-b** [comment]
> Please add Tested by section - and explain this corner case.

done

File: include/dump_offload.hpp [hunk 1]
```diff
@@ -180,6 +212,7 @@ class Handler : public std::enable_shared_from_this<Handler>
             unixSocket.close();
             std::remove(unixSocketPath.c_str());
         }
+        resetOffloadURI();
```
Comment (gtmills) [inline include/dump_offload.hpp:215]: Don't need to do this more often? What if you hit line 202?

        if (ec)
        {
            BMCWEB_LOG_ERROR << "Unix socket file clean up failed "
                             << unixSocketPath;
            this->connection->sendStreamErrorStatus(
                boost::beast::http::status::internal_server_error);
            return;
        }
        
        
        
I guess shouldn't happen

Comment (gtmills) [inline include/dump_offload.hpp:215]: Don't need to do this more often? What if you hit line 202?

        if (ec)
        {
            BMCWEB_LOG_ERROR << "Unix socket file clean up failed "
                             << unixSocketPath;
            this->connection->sendStreamErrorStatus(
                boost::beast::http::status::internal_server_error);
            return;
        }
        
        
        
I guess shouldn't happen

Comment (raviteja-b) [reply include/dump_offload.hpp:215]: yes it should not happen unless /tmp file system screwed up
in that case system reboot needed to recover

**Review by gtmills — APPROVED** [review]
